person.rb

```diff
@@ -1,15 +1,16 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :id, :name
+  attr_reader :rentals, :name
 
   def initialize(id, name)
     super()
     @id = id
     @name = name
+    @rentals = [] # To store rentals
   end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍